src/components/partnerRequestList/partnerRequestList.js

```diff
@@ -277,12 +281,17 @@ function PartnerRequestList({}) {
     };
   }, []);
 
+  const handleCheckboxClick = (params) => {
+    const selectedRows = params.api.getSelectedRows();
+    setMessage(selectedRows?.length);
+  };
+
   const handleReject = () => {
-    setRowData(rowData);
+    alert(message ? `${message} Partner Accounts selected for Reject` : "");
```

Comment:
remove alerts, just keep console log for now

Reply:
done with commnts